QuadPaySDK/PaymentWidget/PaymentWidget.swift

```diff
@@ -107,8 +107,11 @@ extension PaymentWidget {
         stackView.addArrangedSubview(paymentWidgetHeaderText)
         stackView.addArrangedSubview(paymentWidgetSubText)
         stackView.addArrangedSubview(timelapseGraphView)
+        stackView.addArrangedSubview(feeTierView)
         if(maxFee != 0){
-            stackView.addArrangedSubview(feeTierView)
+            feeTierView.isHidden = false
+        }else{
+            feeTierView.isHidden = true
```

Comment:
Can this not be changed to 

`feeTierView.isHidden = maxFee != 0`

Reply:
I tested it and is not hiding the view.